mne/viz/_3d.py

```diff
@@ -382,27 +382,23 @@ def plot_evoked_field(evoked, surf_maps, time=None, time_label='t = %0.0f ms',
         # Make a solid surface
         vlim = np.max(np.abs(data))
         alpha = alphas[ii]
-        with warnings.catch_warnings(record=True):  # traits
-            renderer.surface(surface=surf, color=colors[ii],
-                             opacity=alpha)
+        renderer.surface(surface=surf, color=colors[ii],
+                         opacity=alpha)
 
         # Now show our field pattern
-        with warnings.catch_warnings(record=True):  # traits
-            renderer.surface(surface=surf, vmin=-vlim, vmax=vlim,
-                             scalars=data, colormap=colormap)
+        renderer.surface(surface=surf, vmin=-vlim, vmax=vlim,
+                         scalars=data, colormap=colormap)
 
         # And the field lines on top
-        with warnings.catch_warnings(record=True):  # traits
-            renderer.contour(surface=surf, scalars=data, contours=21,
-                             vmin=-vlim, vmax=vlim, opacity=alpha,
-                             colormap=colormap_lines)
+        renderer.contour(surface=surf, scalars=data, contours=21,
+                         vmin=-vlim, vmax=vlim, opacity=alpha,
+                         colormap=colormap_lines)
 
     if '%' in time_label:
         time_label %= (1e3 * evoked.times[time_idx])
-    with warnings.catch_warnings(record=True):  # traits
-        renderer.text(x=0.01, y=0.01, text=time_label, width=0.4)
-        with SilenceStdout():  # setting roll
-            renderer.set_camera(azimuth=10, elevation=60)
+    renderer.text(x=0.01, y=0.01, text=time_label, width=0.4)
+    with SilenceStdout():  # setting roll
```

Comment:
The `SilenceStdout` should also live in the Mayavi backend, it's to deal with a peculiarity of (an old version of) that library